test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py

```diff
@@ -36,6 +36,14 @@ def step(context, new_community_channel_name):
 @Then("the user lands on the community channel named |any|")
 def step(context, community_channel_name): 
     _statusCommunityScreen.verify_channel_name(community_channel_name)
+    
+@When("the admin creates a community category named |any|, with channel |any| and with the method |any|")
+def step(context, community_category_name, community_channel_name, method):
+    _statusCommunityScreen.create_community_category(community_category_name, community_channel_name, method)
+    
+@Then("the user lands on the community category named |any|")
+def step(context, community_category_name): 
+    _statusCommunityScreen.verify_category_name(community_category_name)
```

Comment:
Maybe later we can improve this verification so that it checks the channel is part of the category as well

Reply:
Yes, i still trying to do it without recursive child search but i anyway need this for editing and deleting categories